Dockerfile.deploy

```
@@ -0,0 +1,16 @@
+FROM node:18
+LABEL org.opencontainers.image.source https://github.com/eddiehubcommunity/LinkFree
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+ARG NODE_ENV
+RUN if [ "$NODE_ENV" = "development" ]; then npm install --ignore-scripts; else npm install  --omit=dev --ignore-scripts; fi
```

We can remove the conditional, because we will not run the file via Compose

```suggestion
RUN npm install  --omit=dev --ignore-scripts
```